docs/tutorial/dapr-ambient-and-knative.md

```diff
@@ -0,0 +1,201 @@
+## Dapr Ambient :: Step by step Tutorial
+
+In this short tutorial we will install Dapr Ambient and configure a set of applications to work with it.
+
+We will be using Knative Serving and Knative Functions to demonstrate where Dapr Ambient can add value for serverless scenarios. 
+
+## Prerequisites and installation 
+
+We will be creating a local KinD cluster where we will install Knative Serving and Dapr.
+
+For this you will need to install the following CLIs:
+
+- [Install `kubectl`](https://kubernetes.io/docs/tasks/tools/)
+- [Install `kind`](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [Install `helm`](https://helm.sh/docs/intro/install/) 
+- [Install `docker`](https://docs.docker.com/engine/install/)
+- [Install the Knative Functions `func` CLI](https://knative.dev/docs/functions/install-func/)
+- [Install the `dapr` CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+
+Create a local Kubernetes cluster with: 
+
+```
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
```

Comment:
@mcruzdev `kind create cluster` should work now.. as we are not going to use Knative